sample/kotlin/src/main/kotlin/com/datadog/android/sample/sessionreplay/ViewGroupComponentsFragment.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.sessionreplay
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.datadog.android.sample.R
+
+internal class ViewGroupComponentsFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_view_group_components, container, false)
+    }
+}
```

Comment:
minor: can be pretty much simplified

```suggestion
internal class ViewGroupComponentsFragment : Fragment(R.layout.fragment_view_group_components)
```

Reply:
Should we keep the `onCreateView` function because probably in the future we need to setup some views programatically in this function?

Reply:
sure, you can keep it in this case, it is not a blocking comment anyway